integers/overflow/05_simple_goto.c

```diff
@@ -21,7 +21,7 @@ void integer_overflow_goto(int x) {
         printf("%i\n", val);
     }
 
-    val = val * 2;
+    val = val * 2u;
 
     if(i == 0) {
         i++;
```

Comment:
I don't think that I meant for this line to cause any kind of overflow. It only needs to be executed once to cause an overflow in `printf("%i\n", val * 3);` so placing the line in the last if statement seems to be a better idea.
```suggestion
    if(i == 0) {
        i++;
        val = val * 2;
```

Reply:
This does seem to do the trick !
Happy to have been able to better the benchmark, even so slightly.

Again, thank you for your time and for your quality test suite, best regards.
Dario Pinto, intern in the FramaC team ! :)